src/sentry/static/sentry/app/components/links/listLink.tsx

```diff
@@ -10,16 +10,15 @@ type DefaultProps = {
   activeClassName: string;
 };
 
-type Props = DefaultProps & {
-  to: LocationDescriptor;
-  className?: string;
-  query?: string;
-  onClick?: () => void;
-  // If supplied by parent component, decides whether link element
-  // is "active" or not ... overriding default behavior of strict
-  // route matching
-  isActive?: (location: LocationDescriptor, indexOnly?: boolean) => boolean;
-};
+type Props = DefaultProps &
+  React.ComponentProps<typeof Link> & {
```

Comment:
Please...are you sure about making other props available here, like href?

Reply:
Yes, that is the reason for this change :)